config/initializers/sentry.rb

```diff
@@ -0,0 +1,4 @@
+Sentry.init do |config|
```

Comment:
Do you want some checks around this so that you are only logging Sentry errors if Rails.env is production? VCD has if Rails.env.eql?('production') && ENV['SENTRY_DSN'].present?

Reply:
I want to log Sentry errors on all environments, so I might use only `ENV['SENTRY_DSN'].present?`

Reply:
I believe the Rails.env.eql?('production') check is to ensure sentry errors aren't raised when Rails.env is test or development, but as there isn't a SENTRY_DSN set it in that case it probably doesn't matter.